src/lib/PPrint.hs

```diff
@@ -464,20 +473,28 @@ instance Pretty (SolverBinding n) where
 
 instance Pretty (Binding s n) where
   pretty b = case b of
-    AtomNameBinding   info -> "Atom name:" <+> pretty info
-    DataDefBinding    dataDef -> pretty dataDef
-    TyConBinding      dataDefName e -> "Type constructor:" <+> pretty dataDefName <+> (parens $ "atom:" <+> p e)
-    DataConBinding    dataDefName idx e ->
-      "Data constructor:" <+> pretty dataDefName <+>
-      "Constructor index:" <+> pretty idx <+> (parens $ "atom:" <+> p e)
-    ClassBinding    classDef    -> pretty classDef
-    InstanceBinding instanceDef -> pretty instanceDef
-    MethodBinding className idx _ ->
-      "Method" <+> pretty idx <+> "of" <+> pretty className
-    ImpFunBinding f -> pretty f
-    ObjectFileBinding _ -> "<object file>"
-    ModuleBinding  _ -> "<module>"
-    PtrBinding     _ -> "<ptr>"
+    AtomNameBinding   info
+      -> "Atom name:" <+> pretty info
+    DataDefBinding    dataDef
+      -> pretty dataDef
+    TyConBinding      dataDefName e
+      -> "Type constructor:" <+> pretty dataDefName <+> (parens $ "atom:" <+> p e)
+    DataConBinding    dataDefName idx e
+      -> "Data constructor:" <+> pretty dataDefName <+> "Constructor index:" <+> pretty idx <+> (parens $ "atom:" <+> p e)
+    ClassBinding    classDef
+      -> pretty classDef
+    InstanceBinding instanceDef
+      -> pretty instanceDef
+    MethodBinding className idx _
+      -> "Method" <+> pretty idx <+> "of" <+> pretty className
+    ImpFunBinding f
+      -> pretty f
+    ObjectFileBinding _
+      -> "<object file>"
+    ModuleBinding  _
+      -> "<module>"
+    PtrBinding     _
+      -> "<ptr>"
```

Comment:
This reads a bit weirdly to me. Is there a meaningful change you've made here, or is this purely a reformatting? If the latter, please revert it to what it used to be.

I don't want to nitpick formatting, but this is pretty inconsistent with what we've been using so far, and I'd rather mark formatting-only changes explicitly.